streams/in.m3u

```diff
@@ -16,7 +16,7 @@ https://sneh-z5api.herokuapp.com/?c=0-9-tvpictureshd
 #EXTINF:-1 tvg-id="AndTVHD.in" status="error",&TV HD (720p) [Geo-blocked]
 https://sneh-z5api.herokuapp.com/?c=0-9-tvhd_0
 #EXTINF:-1 tvg-id="ANEWSdonsTVBhangraFlava.in" status="timeout",A NEWSDONs TV- SWAG SADDA VAKRAA (720p) [Not 24/7]
-http://newsjatt.camdvr.org:1935/newsjatt/myStream/playlist.m3u8
+http://tiny.cc/jattvlink
```

Comment:
First of all, the broadcast hangs on this frame:
<img width="640" alt="image" src="https://user-images.githubusercontent.com/7253922/159679205-8977f290-fdeb-45b9-ad15-2ae5471b83c4.png">

Second, it is better to avoid unnecessary redirects and use the original link
```suggestion
https://video-weaver.maa01.hls.ttvnw.net/v1/playlist/CsYF2GoZykCX_x0zY0tndGM3IIgZIZ1JfwrcY5wTw4Q4W-hkMZ054HL7yeY32S_zsJ2xVR9d2xbVA9vdEYlYkwBgUTmQP7JAB8XnWvqJqOM_BFDrIMFWcNoIOUmbpl9GF8q27TVgBAlDiYlH81IgfSOHCg42IxUZFqCoRpixqTzUPW8plKOtVDK8iLhtjtoTfwhizW5wSQS8PKZf3Jdh47bYCy-jbugLFbOq4Y9jrSLJEjBVd8sIBC3kfoizUzfiPQ7t3dUB5KnUDuU3RPey-HbM0PnVgxUa5aNWLHV88nRrw82qjMh2cZSL9EdR0TqPgk0ic7ydINyrp9vNZa_hZWzuT__AcLQrN8OqEesbfQcUNEQMgdf3qFqsS2bwLn5sF6kjPZb9TP7JTLjrBQycJE9L1pHIPJOGDUngHiN-MOXBWa1TvhtSPgHh98kUMSueQb67XvVLwtBp4PQ2cvEj6cb0ChzJ5Yecw2tWoNqAvOnlZi_Gy3nW7DsIWB6H9EMko53Pcd4YjfqCko5vU-y7oWSRe8o7gUkIN0ZFj__fxGqwQVhe3adPXf2owxDLWf-xo1laQf-DGiEUjauVzb6prG9SSbT6E_mcmrtMdh4maYZflIfjf3jwut2kVOeu-NOKhuhl0jS1kNRnmDTzjdA4I5U5c-p7JwqnwAxB0A4XJy9DczgGVsqj4qzqJwaonm9V4Ogqas2CrueHzqCWwXjCq4rbpmrdc7ho-_Vt9e1UX93IFym-6gzg2IHGMPd_tuDxM-tm5GMyKLAVNteIx9uHw5iPjoYmYl4MRUZ89KobqTsGMUQPRHQbShs434OC9ZMyYeTdtoy1nwvTbINWQ3SSggwzz0f8ES3CUajGJzNFo2uKNE2cvGzETeOHLOiL2-g0mqroNqe3yRUFe85FlLOy7v3RyyoRRuwyAQ2TUxtQqNd6JT8Oz-4-B6oaDIBPuEF4rwud0pIZbSABKgl1cy1lYXN0LTEw3AE.m3u8
```

Reply:
Sounds like it's using free.fullspeed, but hiding it in a bit.ly to gain profits from it basically.